src/styl/layouts/support-form.styl

```diff
@@ -4,27 +4,122 @@
   display: block
   .support-form__inputs
     opacity: 0
-    transition: opacity var(--transition-speed-medium)
   &--active
     .support-form__inputs
       opacity: 1
 
-.support-form__button, .support-form__custom-input, .support-form__submit
-  color: var(--color-red)
+.support-form__schedule-option-right
+  border-radius:  0px 4px 4px 0px;
+
+.support-form__schedule-option-left
+  border-radius:  4px 0px 0px 4px;
+
+.support-form__button
+  &:hover:not(:focus):not(.ui-button--active)
+      color: var(--color-white-soft-transparent)
+      border-color: var(--color-white-soft-transparent);
+  &.ui-button--active
+    color: var(--color-black-soft);
+    background-color: var(--color-white-soft);
+    border-color: var(--color-white-soft);
+
+.support-form__input-wrapper
```

Comment:
if you just need this as a helper and no other styles on this element this is a perfect thing to add to the stylus library. I've documented a flow for how to do that while building out features here: https://github.com/novaramedia/nm-stylus-library?tab=readme-ov-file#releases